cfgov/jinja2/v1/owning-a-home/index.html

```diff
@@ -0,0 +1,303 @@
+{% extends "base.html" %}
+{% set active_page = "home" %}
+
+{% block title %}Owning a Home{% endblock title %}
+
+{% block content %}
+<main class="index content content__2-1 content__bleedbar" id="main" role="main">
+
+    <section class="content_hero">
+
+       <!-- HERO -->
+      <div class="content-l content_wrapper content_no-sidebar">
+
+      <div class="content-l_col content-l_col-2-3">
+          <h1 class="page-title">Owning a Home</h1>
+          <h2>We're here to help you with buying and financing your home. Learn what to expect and what questions to ask, and get tools and resources to help you make decisions.</h2>
+
+        </div>
+        <div class="content-l_col content-l_col-1-3 hero-illu-inner">
+            <img class="hero-illu" src="{{url_for('static', filename='owning-a-home/img/hero-housekey.png')}}" alt="Illustration of house keys">
+        </div> <!-- /.col-4 -->
+      </div>
+
+    </section>
+
+    <div class="content_wrapper">
+        <section class="content_main">
+
+          <section class="section__intro content-l">
+            <div class="content-l_col content-l_col-1">
+              <h2>Know the process</h2>
+              <p class="u-mb0">Buying a home is exciting, but the process can feel overwhelming at times. Our guide lets you know what to expect and what to do next. Our tools and resources will help you every step of the way.</p>
+            </div>
+          </section>
+
+          <section class="section__phase block__border-top block__padded-top block__sub">
+
+            <div class="content-l">
+              <figure class="phase_illu content-l_col content-l_col-1-4">
+                <img src="{{ }}" alt="Illustration of the home-buying process">
+              </figure>
+
+              <div class="phase_intro content-l_col content-l_col-1-2">
+                <div class="content-l_col content-l_col-1 content-l__large-gutters">
+                <h3 class="u-mb15"><a href="{{ base_url }}process/prepare" class="heading_link" id="prepare-header-link">Prepare to shop</a></h3>
+                <p>Not sure how to get started, how much you can afford, or what to expect when buying and financing a home? Set yourself up for success with a little bit of preparation.</p>
+                <p>
+                  <a class="jump-link jump-link__right" href="{{ base_url }}process/prepare" id="prepare-learn-link">
```

Comment:
This file was removed inhttps://github.com/cfpb/cfgov-refresh/pull/3165/commits/adea54f8ecb4f983b08882aaf842c2e812e402bf1.